app/uk/gov/hmrc/transitmovementsconverter/models/ModelHelpers.scala

```diff
@@ -1,372 +0,0 @@
-/*
```

Comment:
For reference, this has become `ModelImplicits`, but Git thinks its a removal because of many changes occurring.